spec/models/form526_submission_spec.rb

```diff
@@ -262,6 +262,17 @@
           exhausted_backup
         )
       end
+
+      it 'handles the edge case where an sub succeeds during query building' do
```

Comment:
Nit: if the spirit of this test is to ensure a submission does _not_ end up in the query results, I might structure it differently and do `.not_to contain(expired)` or whatever the syntax is for that, instead of expecting all that is included. It seems more explicit and easier to maintain going forward. At least for this edge case, I can understand why you structured the rest of the tests the way you did.

Reply:
that's a good note, but i wanted to keep the paradigm simple. This way for each scope i test what it includes. My thinking was that if I test for everything it doesn't include the spec just becomes kind of messy. But yes, best practice would probably agree with you.